image/scanner/rhel/Dockerfile

```diff
@@ -24,13 +24,13 @@ COPY --from=extracted_bundle /bundle/scanner ./
 
 COPY ./THIRD_PARTY_NOTICES/ /THIRD_PARTY_NOTICES/
 
-RUN dnf upgrade -y && \
-    dnf install -y ca-certificates xz && \
-    dnf clean all && \
+RUN microdnf upgrade && \
+    microdnf install xz && \
```

Comment:
`ca-certificates` comes pre-installed, so it's not needed here.

Reply:
Not related to this PR but if we switch to pure golang implementation of xz we can remove whole line https://github.com/ulikunitz/xz